Update dependencies

albertborsos opened the pull request:
Update `sentry/sentry` to `~1.8` and `raven-js` to `~3.17`.

Resolve https://github.com/hellowearemito/yii2-sentry/issues/17

albertborsos committed:
update dependencies versions to the latest versions

albertborsos committed:
move fxp-asset installer paths from `extra` to `config`

because it is deprecated

albertborsos committed:
update sentry and raven-js

update sentry to 1.6.2

albertborsos committed:
update composer-asset-plugin to ~1.3 for travis

albertborsos committed:
use trusty to support hhvm builds

albertborsos committed:
update sentry/sentry to ~1.7.0

albertborsos committed:
fix mocked Raven_Client

because `Raven_Client::close_curl_method` runs on destruct since sentry:1.7

albertborsos committed:
call Mockery::close() in tearDown methods of tests

thiagotalma commented on composer.json:
```diff
@@ -30,9 +30,12 @@
-        "asset-installer-paths": {
-            "bower-asset-library": "vendor/bower"
+    "config": {
+        "fxp-asset": {
```
Hello @albertborsos!
It is worth remembering that the Yii2 officially abandoned the fxp-asset.
https://github.com/yiisoft/yii2-app-advanced/blob/master/composer.json#L35

thiagotalma replied:
https://github.com/yiisoft/yii2/commit/d3f97f7e64f9ced72a418b70d4e2ad9f3f23d902
https://github.com/yiisoft/yii2-app-advanced/commit/4a892a65c122bcea294fd93fc4fb860b2789dd58

albertborsos replied:
Thanks for the information, I will update it!

nkovacs replied:
Only the root package's config is used, so this is only used when working on this package.

Alex-Bond commented:
Will you merge it anytime soon?

albertborsos committed:
composer update

albertborsos committed:
run travis builds on php7.1 machine too

nkovacs reviewed (approved):
LGTM

nkovacs commented:
Codesniffer is failing though, you need to specify a version (probably 2.x). The coding standards package allows 3.x and it probably broke it.

albertborsos committed:
fix builds by mito/yii2-coding-standards update

and reverting codesniffer to 2.9.1 because from version 3.0 the folder structure is different

albertborsos committed:
composer update

albertborsos committed:
set yii2 version constraint to ~2.0.0

because yii2 2.1 coming soon

albertborsos commented:
Thank you for your patience. I released it under version `1.0.3`
